### ISSUE TITLE
BlockGrad and weight decay on backpropagation

### ISSUE THREAD

**Issue by cainiao2hao** [apache/incubator-mxnet#7289]
I have a symbol for training like this. 
```python
mx.sym.Group([loss0, loss1, mx.symbol.BlockGrad(output0)])
```
The output0 will be used to calculate some accuracy metric. But it seems like 'BlockGrad' will also affect network weight because of the weight decay. Is there any way to ignore the output0 part when doing backpropagation but also use the output0 to evaluate the training result?
@piiswrong @mli

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
Also, do please check out our [forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) for general "how-to" questions.
